Smith branch

CasMaul opened the pull request:
[InstallationErrros.zip](https://github.com/rylagek/pack_everything/files/8254308/InstallationErrros.zip)
attached screenshots of errors for some OS's. SSH connection issue for Ubuntu 20.04 desktop and server.

CasMaul committed:
Update README.md

added notes section for future documentation and questions that will be answered in notes section when received

CasMaul committed:
Update README.md

CasMaul committed:
Update UbuntuDesktop1804

New ISO link is the correct one. Successful build after this change

CasMaul committed:
Update UbuntuServer2004

ISO link is updated. Machine is created but no artifact made yet. Still missing scripts so they are commented out

CasMaul committed:
Update ubuntuServer2004

Updated ISO link and checksum.  Error occurs in VM so no artifact yet

CasMaul committed:
Update windows10

replaced checksum value with correct hash. Machine fully operational

CasMaul committed:
Create README

README file  is created within vbox for continuous documentation.

CasMaul committed:
Update README and Ubuntu Builds

cleaned up ssh code in ubuntu pkr.hcl files. Added known issues section in README

rylagek commented on README.md:
```diff
@@ -71,3 +71,13 @@ Generic [Packer](https://www.packer.io/docs) Templates with [chained builders](h
+How to run: Navigate to proper directory that contains file "packer"
+run the command: `packer build packer` and it should build the vm in that folder assumning there are `.pkr.hcl` files in the directory
+
+## Questions:
```
Is this section meant to be a FAQ?

rylagek commented on README.md:
```diff
@@ -71,3 +71,13 @@ Generic [Packer](https://www.packer.io/docs) Templates with [chained builders](h
+run the command: `packer build packer` and it should build the vm in that folder assumning there are `.pkr.hcl` files in the directory
+
+## Questions:
+1. How to run `.pkr.hcl` on Windows and Linux?
```
Packer commands are the same across operating systems - running `packer` without additional arguments will list all possible commands

rylagek commented on README.md:
```diff
@@ -71,3 +71,13 @@ Generic [Packer](https://www.packer.io/docs) Templates with [chained builders](h
+## Questions:
+1. How to run `.pkr.hcl` on Windows and Linux?
+2. Where can I find Windows examples or documentation in vbox hcl?
+3. Is there documentation anywhere for why certain configurations were made?
```
Please be more specific

rylagek commented on README.md:
```diff
@@ -71,3 +71,13 @@ Generic [Packer](https://www.packer.io/docs) Templates with [chained builders](h
+1. How to run `.pkr.hcl` on Windows and Linux?
+2. Where can I find Windows examples or documentation in vbox hcl?
+3. Is there documentation anywhere for why certain configurations were made?
+4. Can we copy Packer resources into this repo from Vater?
```
The intent is to use this repo to feed VATER. That being said, I will absolutely add vSphere-iso builds once I complete my current dev sprint

rylagek commented on builders/vbox/README.md:
```diff
@@ -0,0 +1,48 @@
+checksum. If this occurs, verify the ISO link if correct along with the
+checksum value. You may even download an ISO to reference locally.
+
+2. When testing a build you may find that it will finish in under 10 seconds with
```
A good practice when building and encountering the option of using `--force`, first consider using `-var` and changing the artifact location so if your new build fails you aren't deleting your current "best" build

rylagek reviewed (changes requested):
Looks pretty good! Once the conversations are resolved, I'll approve

CasMaul committed:
Resolved Pull Request Changes

Any part mentioned in the requested changes of my pull request are fixed in this commit

rylagek reviewed (approved):
👍
